### PR TITLE
Add FakeQuantize op to ONNX importer

### PR DESCRIPTION
**Pull request by mitruska**
Add FakeQuantize op (not onnx standard) to the importer.

### CODE REVIEW

File: ngraph/test/models/onnx/quantization/fake_quantize_from_mobilenet.prototxt [hunk 1]
```diff
@@ -0,0 +1,70 @@
+ir_version: 6
+producer_name: "pytorch"
+producer_version: "1.5"
+graph {
+  node {
+    output: "666"
```
Comment (arogowie-intel) [inline ngraph/test/models/onnx/quantization/fake_quantize_from_mobilenet.prototxt:6]: Really ? :)

File: ngraph/test/onnx/onnx_import.in.cpp [hunk 2]
```diff
@@ -2106,6 +2106,67 @@ NGRAPH_TEST(${BACKEND_NAME}, onnx_model_scatter_elements_import_only)
...
+    auto test_case = ngraph::test::NgraphTestCase(function, "${BACKEND_NAME}");
+    test_case.add_input<float>(input_data);
+    test_case.add_expected_output<float>(
+        data_shape, std::vector<float>{2.f,  2.f,  2.f,  2.f,  2.f,   5.5f,  5.5f,  5.5f,
+                                       5.5f, 9.f,  9.f,  9.f,  12.5f, 12.5f, 12.5f, 12.5f,
+                                       16.f, 16.f, 16.f, 16.f, 16.f,  16.f,  16.f,  16.f});
+    test_case.run();
+}
+
+NGRAPH_TEST(${BACKEND_NAME}, onnx_model_fake_quantize_nonconst_intputs_infer)
```
Comment (arogowie-intel) [inline ngraph/test/onnx/onnx_import.in.cpp:2141]:
```suggestion
NGRAPH_TEST(${BACKEND_NAME}, onnx_model_fake_quantize_nonconst_inputs_infer)
```

File: ngraph/test/runtime/ie/unit_test.manifest [hunk 3]
```diff
@@ -161,6 +161,8 @@ onnx_model_conv_with_dynamic_batch
 
 # Segmentation fault
 bool_init_and
+onnx_model_fake_quantize_const_inputs_infer
+onnx_model_fake_quantize_nonconst_intputs_infer
```
Comment (arogowie-intel) [inline ngraph/test/runtime/ie/unit_test.manifest:165]:
```suggestion
onnx_model_fake_quantize_nonconst_inputs_infer
```

File: ngraph/test/models/onnx/quantization/fake_quantize_from_mobilenet.prototxt (hunk 1 above)
Comment (mitruska) [reply ngraph/test/models/onnx/quantization/fake_quantize_from_mobilenet.prototxt:6]: I cut this fragment from a bigger model, without changing any names :)

**Comment by mitruska** [comment]
> LGTM
> 
> But I do not understand what does this mean:
> 
> ```
> // TODO Change the domain
> m_map[""]["FakeQuantize"].emplace(1, op::set_1::fake_quantize);
> ```

Empty domain `""` is default for ONNX standard operators, so we consider other domain for FakeQuantize because it's not a part of the standard. This line actually means the same what `REGISTER_OPERATOR` macro, but the macro uses only `""` domain.
So I wanted to separate FQ op from the standard ops to make changing the domain easier in the future.
Anyway, if we want to keep FakeQuantize in the ONNX standard `""` domain, I can use the macro.

**Pull request merged**